stream_alert/apps/config.py

```diff
@@ -273,9 +275,20 @@ def _determine_last_time(self, date_format):
         return self.last_timestamp
 
     def _save_state(self):
-        param_value = json.dumps(
-            {self._TIME_KEY: self.last_timestamp, self._STATE_KEY: self.current_state}
-        )
+        """Save the current state in the aws ssm paramater store
+
+        Raises:
+            AppStateError: If the parameter is unabled to be saved
+        """
+        try:
+            param_value = json.dumps({
+                self._TIME_KEY: self.last_timestamp,
+                self._STATE_KEY: self.current_state,
+                self._CONTEXT_KEY: self.context,
+            })
+        except TypeError as err:
+            raise AppStateError('Could not serialise state for name \'{}\'. Error: '
```

Comment:
I like the cultural diversity here :P but could we use the american english version of `serialise` here?

Reply:
Ah yep, good point 👍